src/TileSqliteDatabase_P.cpp

```diff
@@ -77,6 +77,7 @@ bool OsmAnd::TileSqliteDatabase_P::open()
             "   y INTEGER NOT NULL,"
             "   z INTEGER NOT NULL,"
             "   image BLOB,"
+            "   time LONG,"
```

Comment:
I am not sure that always need to create table with column time, instead of use method enableTileTimeSupport